apps/meld/meld.py

```diff
@@ -0,0 +1,44 @@
+from talon import Context, Module, actions
+
+mod = Module()
+ctx = Context()
+
+apps = mod.apps
+apps.meld = """
+os: windows
+and app.name: Visual diff and merge tool
+os: windows
+and app.exe: meld.exe
+"""
+
+ctx.matches = r"""
+app: meld
+"""
+
+
+@ctx.action_class("app")
+class AppActions:
+    def tab_open():
+        actions.key("ctrl-n")
+
+    def tab_previous():
+        actions.key("ctrl-alt-pageup")
+
+    def tab_next():
+        actions.key("ctrl-alt-pagedown")
+
+    def tab_reopen():
+        print("Meld does not support this action.")
+
+
+@ctx.action_class("user")
+class UserActions:
+    def tab_jump(number):
+        if number < 10:
+            actions.key(f"alt-{number}")
```

Comment:
This doesn't appear to work, though the shortcut appears to be correct

Reply:
It works for me on windows 11, But it seems like Meld implements this keyboard shortcut in an unfortunate way:  it seems to work only if the current tab is selected, for example by clicking on the current tab.

Reply:
See here [https://gitlab.gnome.org/GNOME/meld/-/issues/839#note_2077059]()